samples/client-sdk/knocking/index.css

```diff
@@ -0,0 +1,61 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+.wrapper {
+  max-width: 800px;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+}
+.wrapper > p {
+  margin: 4px 0;
+}
+
+input {
+  margin: 8px 0;
+}
+
+ul {
+  padding-inline-start: 20px;
+}
+
+.owner-container,
+.guest-container {
+  display: flex;
+}
+
+.form-container,
+.video-container {
+  flex: 1;
+}
+
+.hide {
+  display: none;
+}
+
+button,
```

Comment:
It'd be nice to add a very simple hover state for the buttons as well

Reply:
will add now!